src/test/unit/specific_feature_tests/dialogs.js

```diff
@@ -41,48 +41,27 @@ module("dialogs-opening_or_not", {setup: prepareUnitTestModule});
  */
 /* jshint latedef: nofunc */
 function dialogTestHelper(args) {
-    // Sinon doesn't allow wrapping of `window.open` with a spy because `open`
-    // doesn't look like a function in these browsers.
-    var sinonCantWrapWindowOpen = jQuery.browser.msie &&
-        jQuery.browser.versionNumber <= 8;
+    var wymeditor = jQuery.wymeditors(0);
```

Comment:
Docstrings and comments are very poor replacements for descriptive names, but are good icing on the cake. `dialogTestHelper` doesn't tell me anything at all about what the function *does*, which means even a cursory look through this module requires the reader to find the definition of `dialogTestHelper` and attempt to answer their own version of the "what does this do?". With `tryDialogWithClickAndKeyboardThenAssert`, I keep the abstraction while reading and can just continue on looking at test cases if I don't need to understand how that function tries the dialog with a click and then a keyboard shortcut and asserts.

There might be a better name, but I think the things needed conveying to understand what this function does:
* It does dialog things
* It uses a click
* It uses a keyboard shortcut
* It runs some assertions on the results

The long name is a sign that maybe we could have broken this up into multiple functions (or perhaps not), but lots of typing isn't a good reason to leave a generic name. This is currently the semantic equivalent of `doTestStuff` :)